pkg/cmd/server/bootstrappolicy/policy.go

```diff
@@ -213,7 +213,16 @@ func GetBootstrapClusterRoles() []authorizationapi.ClusterRole {
 				authorizationapi.NewRule("create").Groups(buildGroup).Resources(authorizationapi.JenkinsPipelineBuildResource).RuleOrDie(),
 			},
 		},
-
+		{
+			ObjectMeta: kapi.ObjectMeta{
+				Name: StorageAdminRoleName,
+			},
+			Rules: []authorizationapi.PolicyRule{
+				authorizationapi.NewRule(readWrite...).Groups(kapiGroup).Resources("persistentvolumes", "persistentvolumes/status", "storageclasses").RuleOrDie(),
+				authorizationapi.NewRule(read...).Groups(kapiGroup).Resources("events", "limitranges", "namespaces", "namespaces/status",
```

Comment:
We are expecting this role as someone who doesn't necessarily need to be a developer or fully understand the administration of the cluster (as you stated, if they do have that knowledge and ability they can be promoted to cluster-admin or given admin role on a specific project or even cluster-reader type of role to help assist)  So I'm thinking about this in terms of an enterprise and in the enterprise there are storage-admins, dba's, etc... they control/own storage assets and want to give them the ability to manage/create the storage resources that can be consumed by the cluster with limited knowledge of the cluster.  So jdoe owns 50 nfs servers, OCP cluster needs access to these servers, so cluster-admin gives jdoe a role of storage-admin so he can properly create the storageclasses, pv's ,etc... Also, some of the read perms I'm thinking are things like being able to see quotas and resource limits as they may impact his storage assets, again without being given full admin or other roles.  Namespaces I was thinking might be useful as a reference as he is creating these classes and objects??  Claims I originally had in there, but removed because they again are project scoped and not sure a storage-admin who simply needs to set up classes and pvs would care about pvcs - besides, they can see what pvcs are bound anyway through the PV.  Maybe my thought process on this role is off?  @erinboyd  want to chime in here?